apps/antalmanac/src/stores/Schedules.ts

```diff
@@ -25,7 +28,12 @@ export class Schedules {
     constructor() {
         const scheduleNoteId = Math.random();
         this.schedules = [
-            { scheduleName: 'Schedule 1', courses: [], customEvents: [], scheduleNoteId: scheduleNoteId },
+            {
+                scheduleName: `${termData[0].shortName.replaceAll(' ', '-')}`,
```

Comment:
After you write the new function for calculating the default schedule name, use that here.

Reply:
For this, there is a problem: The construction of Schedule is inside the AppStore, so Schedule will not be able to get the function from AppStore since it has not been initialized yet.